src/command/graph/check.rs

```diff
@@ -29,6 +32,21 @@ pub struct Check {
     #[structopt(long, short = "s", parse(try_from_str = parse_schema_source))]
     #[serde(skip_serializing)]
     schema: SchemaSource,
+
+    /// Minimum number of requests within the time window for a query to be
+    /// considered
+    #[structopt(long, parse(try_from_str = parse_query_count_threshold))]
+    query_count_threshold: Option<i64>,
+
+    /// Minimum percentage of requests within the time window for a query to be
+    /// considered, relative to total request count. Expected values are 0-5
+    /// (i.e. minimum 5% of total request volume)
+    #[structopt(long, parse(try_from_str = parse_query_percentage_threshold))]
+    query_percentage_threshold: Option<f64>,
```

Comment:
This takes in an int, checks its value is positive, divides by 100, and returns the float percentage value

Reply:
it also checks if the number is greater than 100. we should call out the valid range of values in the help text.

> Expected values are 0-5

^ this doesn't quite make sense to me

Reply:
Yeah, that makes sense. That's what I get for copypasta 😅